packages/venia-concept/src/actions/createAccountPage/__tests__/asyncActions.spec.js

```diff
@@ -0,0 +1,48 @@
+import { dispatch, getState } from 'src/store';
+jest.mock('src/store');
+const thunkArgs = [dispatch, getState];
+
+import { createAccount } from '../asyncActions';
```

Comment:
```suggestion
import { createAccount } from '../index';
```

Turns that index test coverage to 100% ;)